src/Polytopes/LinearProgram.jl

```diff
@@ -0,0 +1,147 @@
+@doc Markdown.doc"""
+    LinearProgram(P, c [, k])
+
+The linear program on the feasible set P (a Polyhedron) with
+ respect to the function x ↦ dot(c,x)+k where k is optional (default 0).
+
+""" struct LinearProgram
+   feasible_region::Polyhedron
+   polymake_lp::Polymake.BigObjectAllocated
+   convention::Symbol
+   function LinearProgram(Q::Polyhedron, objective::AbstractVector, k; convention = :max)
+      P=Polyhedron(Polymake.polytope.Polytope(pm_polytope(Q)))
+      ambDim = ambient_dim(P)
+      size(objective, 1) == ambDim || error("objective has wrong dimension.")
+      lp = Polymake.polytope.LinearProgram(LINEAR_OBJECTIVE=homogenize(objective, k))
+      pm_polytope(P).LP = lp
+      new(P, lp, convention)
```

Comment:
Perhaps also check the value for `convention` to restrict it to `:min`and `:max` (?) a bunch of other code seems to rely on it only having exactly those two values.

Perhaps this would also be something one could/should use enums for (see https://docs.julialang.org/en/v1/base/base/#Base.Enums.Enum)?